### PR TITLE
Project 1 - Mabel

### PR DESCRIPTION
**Pull request by Mabelliaw123**
I have a hard time creating Flexbox, whereby Media Q doesn't work at the end. What I did was, redo ing yet again. 
The changes updated made could be undefined, due to the repetition changes constantly. 
Nevertheless I create a simple website which focused on the code rather than the content itself. 

Mvh/ Mabel

### CODE REVIEW

**Comment by HIPPIEKICK** [comment]
Hi Mabel! Did you deploy your project to netlify? Could you provide a link? :)

**Comment by Mabelliaw123** [comment]
Hi Matilda!

Yes, here it is .
https://compassionate-hodgkin-c24e80.netlify.app/code/

Mvh/Mabel

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,119 @@
...
 
-    <section class="">
-      <!-- The other news  -->
+    <!--small news section one-->
+
+    <section class="text">
+
+        <div class="container">
+            <div class="image"><img src="art market.png"></div>
+            <div class="news-cardthree"><b>
```
Comment (HIPPIEKICK) [inline code/index.html:54]: Try to go with kebab-case for all the class names, for better readability

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Good job on this project Mabel! I would like to ask you to go through your HTML document again and give all nested elements new lines. Take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap :)

Also, take a look at my comment about horizontal scroll. Fix these things and you're good to go!

We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

**Pull request closed without merging**